params/config_arbitrum.go

```diff
@@ -96,6 +96,17 @@ func ArbitrumRollupGoerliTestnetParams() ArbitrumChainParams {
 	}
 }
 
+func ArbitrumRinkebyTestParams() ArbitrumChainParams {
+	return ArbitrumChainParams{
+		EnableArbOS:               true,
+		AllowDebugPrecompiles:     false,
+		DataAvailabilityCommittee: false,
+		// Not used - has init data
+		InitialArbOSVersion: 1,
```

Comment:
Isn't InitialArbOSVersion still used with init data? I'm not sure how else we'd determine the initial ArbOS version. I think we should set it to 3.